src/components/views/donate/OneTime/SelectTokenModal/QRCodeDonation/QRDonationCard.tsx

```
@@ -135,6 +133,10 @@ export const QRDonationCard: FC<QRDonationCardProps> = ({
 			return 'label.sign_into_giveth_for_a_chance_to_win_givbacks';
 		}
 
+		if (!!activeStartedRound) {
```

Biome (1.9.4)

[error] Avoid redundant double-negation.

It is not necessary to use double-negation when a value will already be coerced to a boolean.
Unsafe fix: Remove redundant double-negation


(lint/complexity/noExtraBooleanCast)